scripts/common/common.sh

```diff
@@ -327,11 +327,17 @@ exportKubeconfig() {
     local kubeconfig
     kubeconfig="$(${K8S_DISTRO}_get_kubeconfig)"
 
-    current_user_sudo_group
-    if [ -n "$FOUND_SUDO_GROUP" ]; then
-        chown root:$FOUND_SUDO_GROUP ${kubeconfig}
+    if [ "$CIS_COMPLIANCE" == "1" ]; then
+        chown root:root ${kubeconfig}
+        chmod 444 ${kubeconfig}
```

Comment:
i feel like in this case the `kubeconfig_setup_outro` is no longer accurate since `FOUND_SUDO_GROUP` is no longer necessary for kubectl access.

Reply:
@emosbaugh I added a comment and updated kubeconfig_setup_outro to check for CIS_COMPLIANCE as well. If it's set it won't need FOUND_SUDO_GROUP.

Reply:
thanks. lgtm